utils/eblif.py

```diff
@@ -8,8 +8,8 @@
 ]
 
 sub_level = [
-    "cname",
     "attr",
+    "param",
```

Comment:
Can you explain this change?  Maybe add a comment to global explaining what is going on here?  This change seems suspicious, but presumably there is a good reason for it.

Reply:
First, the variable `sub_level` was not used at all in the code. Now it lists keywords that define parameters / attributes of a cell, which values are to be stored in a dictionary. I've added a comment to the code.

Reply:
> the variable `sub_level` was not used at all in the code

That answers my questions :)